test/app/script-tag-integration.html

```diff
@@ -0,0 +1,49 @@
+<!doctype html>
```

Comment:
Do we want to include this in the hosted demo app? Don't see much of advantage there but we could do it.